pkg/infra/appcontext/user.go

```diff
@@ -46,11 +46,20 @@ func User(ctx context.Context) (*user.SignedInUser, error) {
 	// Find the kubernetes user info
 	k8sUserInfo, ok := request.UserFrom(ctx)
 	if ok {
+		// token := k8sUserInfo.GetExtra()["id-token"]
+		// if len(token) == 1 {
+		// 	return &user.SignedInUser{
+		// 		Name:    k8sUserInfo.GetName(),
+		// 		UserUID: k8sUserInfo.GetName(),
+		// 		IDToken: token[0], // forward this
+		// 		OrgID:   1,        // from audience
+		// 		UserID:  0,        // ????
+		// 	}, nil
+		// }
+
 		for _, group := range k8sUserInfo.GetGroups() {
 			switch group {
-			case k8suser.APIServerUser:
-				fallthrough
```

Comment:
Omg! I didn't realize `fallthrough` doesn't actually check the next case, falling back to its body regardless. Thanks!